doc/source/getting_started/installing.rst

```diff
@@ -0,0 +1,397 @@
+
+If you use `Spyder`_ as integrated development environment (IDE, see `Development enviroment`_), you should also install it in the new conda environment and run it from there.
+Otherwise, Spyder is not able to use the packages in the conda environment:
+
+.. code-block:: bash
+
+    $ conda install spyder
+
+
+Install ODL and its dependencies
+--------------------------------
+Install ODL and all its (minimal) dependencies by running
+
+.. code-block:: bash
+
+    $ conda install -c odlgroup odl
+
+To skip the ``-c odlgroup`` option in the future, you can permanently add the ``odlgroup`` conda channel (see `Managing conda channels`_):
+
+.. code-block:: bash
+
+    $ conda config --append channels odlgroup
+
+After that, ``conda install odl`` and ``conda update odl`` work without the ``-c`` option.
+
+**Extra dependencies**
+
+- Image and plot displaying capabilities:
+
+  .. code-block:: bash
+
+    $ conda install matplotlib
+
+- Faster FFT back-end using FFTW (currently not in mainstream conda):
+
+  * Install the `FFTW`_ C library version 3 (all possible precisions).
+    Use your Linux package manager for this task or consult the `Windows <http://fftw.org/install/windows.html>`_ or `MacOS <fftw.org/install/mac.html>`_ instructions, respectively.
+
+  * Run
+
+    .. code-block:: bash
+
+        $ pip install pyfftw
+
+- Wavelet transforms (currently not in mainstream conda):
+
+  .. code-block:: bash
+
+    $ pip install pywavelets
+
+- Simple backend for ray transforms:
+
+  .. code-block:: bash
+
+    $ conda install scikit-image
+
+- To run unit tests:
+
+  .. code-block:: bash
+
+    $ conda install pytest
+
+
+
+
+Alternative 2: Installing a release using ``pip``
+=================================================
+
+TL;DR
+-----
+- Install `pip`_
+- Install `FFTW`_
+- Install ODL and dependencies:
+
+  .. code-block:: bash
+
+    $ pip install odl[show,fftw,pywavelets,scikit,testing]
+
+Install a Python interpreter
+----------------------------
+Open a terminal and type ``python`` + Enter.
+If a Python prompt appears, you already have an interpreter installed and can skip this step.
+Otherwise, you need to install it.
+
+**On Linux:**
+
+In the unlikely event that Python is not installed, consult your distro package manager.
+
+**On MacOS:**
+
+Get the latest release (2 or 3) for MacOS `here <https://www.python.org/downloads/mac-osx/>`_ and install it.
+
+**On Windows:**
+
+Installers can be downloaded from `this link <https://www.python.org/downloads/windows/>`_.
+You should pick the latest release for your favorite version (2 or 3).
+
+
+Install ODL and its dependencies
+--------------------------------
+You need to `install pip`_ to be able to install ODL and its dependencies from the `Python Package Index`_ (PyPI).
+Check if running ``pip`` (alternatively: ``pip3``) shows a help message or an error.
+
+For basic installation without extra dependencies, run
+
+.. code-block:: bash
+
+   $ pip install --user odl
```

Comment:
This will fail on windows with the python above (to my knowledge) quite sure the dependencies to build numpy are not available.